vue2-vuetify/src/editor/DateTimeControlRendererEditor.vue

```diff
@@ -237,6 +238,58 @@ const controlRenderer = defineComponent({
         return null;
       }
     },
+    maxDate(): string {
+      let max = this.control.uischema.options?.maxDate;
+      if (max) {
+        let min = this.control.uischema.options?.minDate;
+        let current = new Date(this.date);
+        max = new Date(max);
+
+        if (current > max) {
+          min ? (min = new Date(min)) : (min = '');
+          if (!min || !(max < min)) {
+            this.date = max.toISOString().split('T')[0];
+          }
+        }
+
+        if (min) {
+          max < min
+            ? (max = min.toISOString())
+            : (max = new Date(max).toISOString());
+          return max;
```

Comment:
change for like this
`max = max < min ? min.toISOString() : max.toISOString();`